src/content/docs/errors-inbox/mobile-tab.mdx

```diff
@@ -0,0 +1,260 @@
+
+
+After you've sorted and filtered your events, you can dig a little deeper into the events that lead up to the crash by examining:
+
+* **Custom breadcrumbs**: Utilize the [Record breadcrumb SDK](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb/) to create custom `MobileBreadcrumb` events. This allows you to log specific application interactions that may be relevant to crash analysis.
+* **Handled exceptions**: Use the recordHandledException SDK methods for iOS and Android to annotate where exceptions are handled in your application. These annotations will automatically populate the crash event trail.
+
+For detailed guidance on enhancing crash event trails with custom data, see [Record breadcrumbs](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb/).
+
+To fully leverage our crash analysis tools, make sure to:
+* Use the mobile SDK to create custom `MobileBreadcrumb` or `MobileHandledException` events. 
+* Enable `MobileRequest` events for capturing network request data.
+
+<img
+    title="group crashes event trail"
+    alt="A screenshot showing the event trail for group crashes in the UI"
+    src={errorsinboxEventTrail}
+/>
+
+<figcaption>
+    INSERT_CAPTION_HERE
+</figcaption>
+
+## Mobile stack trace [#stack-trace]
+
+The stack trace for the crashing thread is displayed beneath the crash details. Each frame within the stack indicates the associated library/package name and method. The agent attempts to highlight the specific stack frame containing the programming error responsible for the crash, if detected.
+
+Understanding Stack Traces:
+
+    * **iOS apps**: Source code method names and line numbers are visible if you have uploaded the corresponding dSYM files for each build.
+    * **Android apps using ProGuard**: Package and method names will be de-obfuscated if ProGuard map uploading is enabled.
+    * **Android native crash reports**: These reports will only display source filenames and relative code offsets.
+
+[Email alerts](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-monitoring-email-notifications/) promptly inform the team of new crashes. 
+
+## Android-native crash reporting [#android-native-reporting]
+
+Starting with [New Relic Android agent version 6.7.0](/docs/release-notes/mobile-release-notes/android-release-notes/android-670), to help track and diagnose native crashes, reporting, and analysis has been enhanced to include signal violations and other faults that occur at the [native code](https://developer.android.com/ndk/guides) level during runtime.
+
+These enhancements include:
+
+   * Native crash reports - signal violations and other crashes reported by the app during runtime, including
+  * Signal 4 (Illegal instruction)
+  * Signal 6 (Abnormal termination)
+  * Signal 7 (Bus error/bad memory access)
+  * Signal 8 (Floating-point exception)
+  * Signal 11 (Segmentation violation/invalid memory reference)
+
+* Native runtime exceptions
```

Comment:
I think a period is missing here. See the deploy.

Reply:
Fixed!